dspace-api/src/test/data/dspaceFolder/assetstore/ads-ex.json

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
Again, as mentioned in other PRs, these tests files should be moved to `dspace-server-webapp` alongside the ITs.